course_discovery/apps/course_metadata/data_loaders/api.py

```diff
@@ -999,32 +1031,23 @@ def _process_response(self, response):
             course_run_key = body['course_id']
             try:
                 body = self.clean_strings(body)
-                course_run = CourseRun.objects.get(key__iexact=course_run_key)
+                course_run = CourseRun.everything.get(key__iexact=course_run_key)
                 course_run.short_description_override = body['excerpt']
                 course_run.full_description_override = body['description']
                 course_run.featured = body['featured']
                 course_run.card_image_url = body['featured_image_url']
                 course_run.slug = body['slug']
-                categories = body['categories']
+                course_run.is_marketing_price = body['price']
```

Comment:
Change field name `is_marketing_price` to `is_marketing_price_set` to describe WP field in better way
<img width="969" alt="Screen Shot 2021-04-29 at 12 09 17 PM" src="https://user-images.githubusercontent.com/5072991/116514122-03852780-a8e4-11eb-8b2e-223b0b2c1a3d.png">

Reply:
Done